client-side/src/components/OuthProvider.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
```

Comment:
Who is the responsilbe to display this OAuthProvider?
(please rename to OAuthProvider)

Reply:
GoogleLogin call it. There is nothing to show only techny

Reply:
GoogleLogin call it. There is nothing to show only techny